supervisor/addons/addon.py

```diff
@@ -149,6 +149,11 @@ def version(self) -> Optional[str]:
         """Return installed version."""
         return self.persist[ATTR_VERSION]
 
+    @property
+    def need_update(self) -> bool:
+        """Return True if an update is available."""
+        return self.version != self.latest_version
```

Comment:
Looking at the UI implementation, we don't show an update if add-on is running detached, should we guard against that her instead?

Reply:
right